src/modules/graphics/Graphics.h

```diff
@@ -145,6 +145,7 @@ class Graphics : public Module
 		FEATURE_INSTANCING,
 		FEATURE_TEXEL_BUFFER,
 		FEATURE_COPY_BUFFER,
+		FEATURE_COMPUTE,
```

Comment:
`FEATURE_GLSL4` covers compute shader availability (after my other comment) so I think a separate enum for compute support isn't needed